spring-session/src/main/java/org/springframework/session/MapSession.java

```diff
@@ -40,7 +40,7 @@
  * @since 1.0
  * @author Rob Winch
  */
-public final class MapSession implements ExpiringSession, Serializable {
+public class MapSession implements ExpiringSession, Serializable {
```

Comment:
I would like to keep it in a distinct PR so we can keep track of why we are making changes.

In regards to getting FST in the Spring IO platform...I believe optional dependencies do not need to be in Spring IO Platform.

cc @wilkinsona